web/src/app/forms/context.ts

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
```

Comment:
Same as above, this belongs in its own PR and converting/refactoring the form system will need to be it's own discussion

Reply:
Will revert. It made updating easier.